docs/gke/configs/deploy.sh

```diff
@@ -76,4 +73,4 @@ kubectl create namespace ${K8S_NAMESPACE}
 kubectl create secret generic --namespace=${K8S_ADMIN_NAMESPACE} admin-gcp-sa --from-file=admin-gcp-sa.json=./${SA_EMAIL}.json
 kubectl create secret generic --namespace=${K8S_NAMESPACE} admin-gcp-sa --from-file=admin-gcp-sa.json=./${SA_EMAIL}.json
 kubectl create secret generic --namespace=${K8S_NAMESPACE} user-gcp-sa --from-file=user-gcp-sa.json=./${USER_EMAIL}.json
-kubectl create secret generic --namespace=${K8S_NAMESPACE} kubeflow-oauth --from-literal=CLIENT_ID=${CLIENT_ID} --from-literal=CLIENT_SECRET=${CLIENT_SECRET}
+kubectl create secret generic --namespace=${K8S_NAMESPACE} kubeflow-oauth-email --from-literal=EMAIL_ADDRESS=${OAUTH_EMAIL_ADDRESS}
```

Comment:
nit this shouldn't be a secret; it can just be a config map since email address doesn't need to be secret.